DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
```

Comment:
Why are the last two separate? Could they not be the same, just rendered slightly differently? Perhaps that's beyond the capabilities of our current CSS (i.e., perhaps it requires us to pull in Sass/LESS to achieve that kind of extensibility). Perhaps we could achieve the same with a little bit of logic in the view to determine where it's being shown - horrid, I know, but would remove the duplication pending a refactor to a better solution. 

How about something like:

`_ManageLoginMethods` - or `_ManageExternalLoginMethods` if adding "external" provides much value.
`_LoginMethodsList`